languages/csharp/exercises/concept/dictionaries/.docs/instructions.md

```diff
@@ -0,0 +1,109 @@
+In this exercise you'll be writing code to keep track of international dialling codes via an international dialing code dictionary.
+
+The dictionary uses an integer for its keys (the dialing code) and a string (country name) for its values.
+
+You have 11 tasks which involve dictionaries.
+
+### 1. Create a New Dictionary
+
+```csharp
+Dictionaries.GetEmptyDictionary();
+// empty dictionary
+```
+
+### 2. Create a Pre-populated Dictionary
+
+Create a dictionary which contains the following 3 dialing codes: "United States of America" which has a code of 1, "Brazil" which has a code of 55 and "India" which has a code of 91:
```

Comment:
Could you explicitly mention what method to implement? Something like:

 ```suggestion
There exists a pre-populated dictionary which contains the following 3 dialing codes: "United States of America" which has a code of 1, "Brazil" which has a code of 55 and "India" which has a code of 91. Implement the (static) `Dictionaries.GetExistingDictionary()` method to return the pre-propulated dictionary:
```

This also applies to the other tasks.